### PR TITLE
feat: Support deployment on PSI lab infra

### PR DESCRIPTION
**Pull request by arajkumar**

### CODE REVIEW

**Review by yzainee-zz — APPROVED** [review]

**Comment by yzainee-zz** [comment]
@arajkumar why not deploying insight serv?

**Comment by arajkumar** [comment]
@yzainee It exceeds the silver quota, also the model bucket names are not in consistent format.

**Comment by yzainee-zz** [comment]
@arajkumar it will be difficult for those working with backbone
recommendations will always fail

File: openshift/helpers.sh [hunk 1]
```diff
@@ -132,8 +123,9 @@ function wait_for_rds_instance_info() {
     while true; do
         echo "Trying to get RDS DB endpoint for ${RDS_INSTANCE_NAME} ..."
 
-        RDS_ENDPOINT=$(get_rds_instance_info | grep -w Address | awk '{print $4}')
-        RDS_ARN=$(get_rds_instance_info | grep -w DBInstanceArn | awk '{print $4}')
+        get_rds_instance_info
```
Comment (prashbnair) [inline openshift/helpers.sh:127]: It is better to use native Linux utilities in shell scripts than relying on json parsing. It is possible that jq may not be installed for everyone.

**Review by prashbnair — CHANGES_REQUESTED** [review]

File: openshift/helpers.sh (hunk 1 above)
Comment (arajkumar) [reply openshift/helpers.sh:127]: Ain't we already rely on non standard binaries like aws-cli, passwd-gen,..etc.?

Comment (prashbnair) [reply openshift/helpers.sh:127]: We can achieve the same without too much complexity using native utilities. We should utilise them as much as possible. This is a POV, I leave it to you.

**Review by prashbnair — APPROVED** [review]

**Pull request merged**